themes/tailwind-audio/template.html

```diff
@@ -0,0 +1,227 @@
+<style>
+  <%- include('/styles.css') -%>
```

Comment:
ejs include here

Reply:
What are the implications of doing this? Like, you couldn't use this template directly with <media-template>, right?

Could we just inline the tailwind CSS in the template.html?

Reply:
it does that, it gets inlined by the build-theme CLI. 
`template.html` is the source file now and the build script creates a `dist/media-theme.html` file and a `dist/media-theme.js` file